.github/workflows/nightly-release.yaml

```diff
@@ -0,0 +1,90 @@
+#
+# Copyright 2022 The GUAC Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This workflow requires a GitHub Personal Access Token named PAT_NIGHTLY_RELEASE
```

Comment:
`PAT_NIGHTLY_RELEASE` is this already setup or need to be?

Reply:
Nope, it needs to be set up.